### PR TITLE
Update n.Makefile

### PR DESCRIPTION
**Pull request by adambraimbridge**
Maybe a bit clearer error message

### CODE REVIEW

File: n.Makefile [hunk 1]
```diff
@@ -78,7 +78,7 @@ _install_scss_lint:
 	@if $(call IS_GIT_IGNORED); then curl -sL https://raw.githubusercontent.com/Financial-Times/n-makefile/$(VERSION)/config/$@ > $@ && $(DONE); fi
 
 .env:
-	@if $(call IS_GIT_IGNORED); then heroku auth:whoami &>/dev/null || (echo Please make sure the Heroku CLI is installed and authenticated by running ‘heroku auth:token’.  See more https://toolbelt.heroku.com/. && exit 1); fi
+	@if $(call IS_GIT_IGNORED); then heroku auth:whoami &>/dev/null || (echo Please make sure the `Heroku CLI toolbelt` is installed - see https://toolbelt.heroku.com/. And make sure you are authenticated by running `heroku login`. && exit 1); fi
```
Comment (matthew-andrews) [inline n.Makefile]: backticks are dangerous

Comment (matthew-andrews) [inline n.Makefile]: also don't update this here, update in `n-makefile` then do `make update-tools`

Comment (matthew-andrews) [inline n.Makefile]: https://github.com/Financial-Times/n-heroku-tools/blob/master/n.Makefile#L1-L2

**Comment by adambraimbridge** [comment]
Thanks @matthew-andrews , will do.

**Comment by adambraimbridge** [comment]
/me notes that this PR was number 404 :)

**Pull request closed without merging**